code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -289,7 +289,10 @@ Pipelines + Other Objects -> Pipe network
 	return list()
 
 /obj/machinery/atmospherics/update_remote_sight(mob/user)

```

Comment:
Test this! Without adding the BLIND flag, I believe the Borer's vision will function as mesons would on a human. The borer sees everything as if it were not inside the vent, plus turf (meson) vision beyond that. The intent is to give them a way to hunt without giving them xray. 

Checking of the user is a borer for everything that enters a vent is not the most optimal way of doing this, however. Borers do need awareness while in a vent, though. Popping up next to a Sec officer the borer had no way of knowing was there is incredibly frustrating. Remember that they are as vulnerable as a newly popped chestburster!